src/products/views/ProductVariant.tsx

```diff
@@ -155,23 +161,28 @@ export const ProductVariant: React.FC<ProductUpdateProps> = ({
     data: ProductVariantUpdateSubmitData,
     variant: ProductVariantDetails_productVariant
   ) => {
-    const isChannelPriceChange = data.channelListings.some(channel => {
+    const channelsHaveChanged = data.channelListings.some(channel => {
       const variantChannel = variant.channelListings.find(
         variantChannel => variantChannel.channel.id === channel.id
       );
       return (
         channel.value.price !== variantChannel?.price?.amount.toString() ||
-        channel.value.costPrice !== variantChannel?.costPrice?.amount.toString()
+        channel.value.costPrice !==
```

Comment:
Can we name these conditions so it's easier to understand what happening here

Reply:
done